Docs/ProjectSpec.md

```diff
@@ -657,6 +657,7 @@ This is a convenience used to automatically generate schemes for a target based
 - [ ] **testTargets**: **[[Test Target](#test-target)]** - a list of test targets that should be included in the scheme. These will be added to the build targets and the test entries. Each entry can either be a simple string, or a [Test Target](#test-target)
 - [ ] **gatherCoverageData**: **Bool** - a boolean that indicates if this scheme should gather coverage data. This defaults to false
 - [ ] **coverageTargets**: **[[Testable Target Reference](#testable-target-reference) - a list of targets to gather code coverage. Each entry can either be a simple string, a string using [Project Reference](#project-reference) or [Testable Target Reference](#testable-target-reference)
+- [ ] **enableGPUFrameCaptureMode**: **GPUFrameCaptureMode** - Property value set for `GPU Frame Capture`. Possible values are `autoEnabled`, `metal`, `openGL`, `disabled`. Default is `autoEnabled`.
```

Comment:
in my testing the values mentioned here are not actually valid, and you have to use the numbers, can you add a test using these strings instead of their numeric values?

Reply:
Do you think this would be covered by this?

https://github.com/yonaskolb/XcodeGen/blob/9422de9925fcd5701aa0b4e12f7b48582b8e2866/Tests/ProjectSpecTests/SpecLoadingTests.swift#L842